microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -295,124 +388,146 @@ See also [DevTools: Focus Mode UI](https://github.com/MicrosoftEdge/MSEdgeExplai
 
 
 <!-- ====================================================================== -->
-<!-- ## DevTools Tooltips -->
+## DevTools Tooltips
 <!-- present in 96, 98 -->
 
+Needs content.
 
 <!-- ====================================================================== -->
-<!-- ## Detached Elements -->
+## Detached Elements
 <!-- present in 96, 98 -->
 
+Memory leaks in web applications are a difficult to locate and repair.
+
+The new Detached Elements panel in Microsoft Edge DevTools helps investigate and resolve a common source of memory leaks.
+
+Memory leaks occur when the JavaScript code of the application retains an increasing number of objects in memory. The JavaScript code should release these objects so that the memory space can be reused by the browser.
+
+<!-- ???more needed, probably with an image, Detached Elements tutorial is in progress??? -->
+
 
 <!-- ====================================================================== -->
-<!-- ## VS Code themes for the DevTools -->
+## VS Code themes for the DevTools
 <!-- present in 96, 98 -->
 
+Needs content.
```

Comment:
Documented in What's New for 93: https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2021/07/devtools#apply-themes-from-visual-studio-code-to-devtools
And here too: https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/customize/theme

I suggest linking to the latter.

Reply:
added the latter